src/styles/components/IMGs/pistolImg.scss

```diff
@@ -0,0 +1,15 @@
+.pistolImg {
```

Comment:
I think that we don't need `img` folder. `pistolImg` could be moved to `styles/components` folder. 
I would rename it to just `pistol`. 
Also, we should name styles files in kebab case to have better readability.